core/src/main/java/io/grpc/ServerCall.java

```diff
@@ -50,6 +53,17 @@
  */
 public abstract class ServerCall<RespT> {
   /**
+   * {@link Attributes.Key} for the remote address of server call.
+   */
+  public static final Attributes.Key<SocketAddress> REMOTE_ADDR_KEY =
+          Attributes.Key.of("io.grpc.RemoteAddr");
```

Comment:
Since it's now essentially just a debug string, I would honestly be okay with "RemoteAddr" and similar, like we are doing for Context.